src/core/SampleBuffer.cpp

```diff
@@ -1196,7 +1201,7 @@ SampleBuffer * SampleBuffer::resample( const sample_rate_t _src_sr,
 		src_data.data_out = dst_buf[0];
 		src_data.input_frames = frames;
 		src_data.output_frames = dst_frames;
-		src_data.src_ratio = (double) _dst_sr / _src_sr;
+		src_data.src_ratio = (double) _src_sr / _dst_sr;
```

Comment:
My Jack is running on 48Khz. I have a click track. I export the project *.wav with 44,1 khz. I insert the *.wav to a sample track. It isn't in sync:
![Bildschirmfoto von 2019-05-29 10-37-38](https://user-images.githubusercontent.com/6502580/58543012-dc58e380-81fe-11e9-9c4f-fee53f266c06.png)

If I revert this line (_dst_sr / _src_sr) it is in sync:

![Bildschirmfoto von 2019-05-29 10-39-19](https://user-images.githubusercontent.com/6502580/58543102-08746480-81ff-11e9-9fee-e08991831025.png)